src/test/java/org/broadinstitute/hellbender/utils/gcs/BucketUtilsTest.java

```diff
@@ -17,7 +17,7 @@
 public final class BucketUtilsTest extends GATKBaseTest {
```

Comment:
Is there some sensible way to unit test the requester-pays support (perhaps using a mock object)?

Reply:
Well what we have here is parsing the command-line arguments, and setting the cloud storage configuration. The former I pretty much already trust. The latter I added a little test for.